### DISCUSSION TITLE
Change Timeout Value

### DISCUSSION THREAD

**Discussion by bijanbina** [rustdesk/rustdesk#5621] [Feature Request] [answered]
I have a poor connection and from time to time the TCP connection drops.
It takes usually 20 seconds for the rust desk to detect that and reconnect. It shows the timeout pop-up and connects automatically which is nice but 20 secs is just too long. Is there any way to change that?
It would also help if you point out where in the code this timeout value is hard-coded

**Comment by dignow** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/blob/master/src/client/io_loop.rs#L203

https://github.com/rustdesk/rustdesk/blob/master/src/server/connection.rs#L610
